apps/tests/Robot/src/robottest/RobotBuilder.java

```diff
@@ -25,7 +25,6 @@
 package robottest;
 
 import com.sun.glass.events.KeyEvent;
-import com.sun.glass.ui.Robot;
```

Comment:
That should be javafx.scene.input.KeyCode;

Additionally, it fails to compile for other reasons:

1. Missing import for javafx.scene.input.MouseButton;
2. Calls to Robot::destroy need to be removed
3. type mismatch in assigning value of Robot::getPixelColor and Robot::getScreenCapture
4. Syntax error -- `new Robot():`